CONTRIBUTING.md

```diff
@@ -130,37 +230,19 @@ This protocol explains how to perform a release of the code on ´master´ branch
 2. In your release branch, update the version number in [`src/multi-vector-simulator/version.py`](https://github.com/rl-institut/multi-vector-simulator/blob/dev/src/multi_vector_simulator/version.py) in  in the format indicated under 1 (commit message: "Bump version number").
 3. Replace the header `[Unreleased]` in the [`CHANGELOG.md`](https://github.com/rl-institut/multi-vector-simulator/blob/dev/CHANGELOG.md)
 with the version number (see 2.) and the date of the release in [ISO format](https://xkcd.com/1179/): `[Version] - YYYY-MM-DD`.
-4. After pushing these changes, create a pull request from `release/vX.Y.Z` towards `master` and merge it into `master`.
+4. After pushing these changes, create a pull request from `release/vX.Y.Z` towards `master` and merge it into `master`. It is normal that git tells you your release is out of date with master, do not hit the "update button"!
 5. Create a [release tag](https://github.com/rl-institut/multi-vector-simulator/releases) on github.
 Please choose `master` as target and use `vX.Y.Z` as tag version. In the description field simply copy-paste the content of the `CHANGELOG`descriptions for this release and you're done!
 For help look into the [github release description](https://help.github.com/en/github/administering-a-repository/creating-releases).
 
-### The actual release
+### The actual release on pypi.org
 *Note*: the point 4 to 7 can be done automatically by running `python prepare_package.py`, they are displayed here explicitly for transparency
```

Comment:
this info does not exist anymore (4 to 7)

Reply:
I explicited these points within the `prepare_package.py` file and I told the user to directly run the script, hence there is no need to know exactly what the script does (it is possible to know this by opening the file though)